.graphqlconfig.yml

```diff
@@ -0,0 +1,20 @@
+projects:
+  test:
```

Comment:
maybe change the name of the project to `chatbot`

Reply:
updated